src/min_heap/tests.rs

```diff
@@ -0,0 +1,130 @@
+use super::{InitError, MinHeap as StableMinHeap};
+use crate::vec_mem::VectorMemory as M;
+use crate::{GrowFailed, Memory};
+use proptest::collection::vec as pvec;
+use proptest::prelude::*;
+use std::cmp::Reverse;
+use std::collections::BinaryHeap;
+use std::fmt::Debug;
+
+#[derive(Debug, PartialEq, Clone)]
+enum Operation<T> {
+    Push(T),
+    Pop,
+}
+
+fn arb_op<T: Clone + Debug>(s: impl Strategy<Value = T>) -> impl Strategy<Value = Operation<T>> {
+    prop_oneof![
+        3 => s.prop_map(Operation::Push),
+        1 => Just(Operation::Pop),
```

Comment:
What is the purpose of the keys `3` and `1` here? Removing them doesn't seem to have any effect.

```suggestion
        s.prop_map(Operation::Push),
        Just(Operation::Pop),
```

Reply:
These numbers are weights affecting the probabilities. I want pushes to be 3x more often so that the heap has a chance to grow relatively large.